ovs/extensions/plugins/tests/alba_mockups.py

```diff
@@ -142,10 +146,53 @@ def _get_data(**kwargs):
     def get_osd_claimed_by(*args, **kwargs):
         """
         Check whether an osd is claimed based on ip and port
-        :return: Alba id or None (only None in our mocked case)
+        :return: Alba id or None
         """
-        _ = args, kwargs
-        # Only called when the osd has not been claimed in the unit test
-        return None
+        _ = args
+        ip = kwargs.get('host')
+        port = kwargs.get('port')
+        if ip is None or port is None:
+            return None
+        return VirtualAlbaBackend.data.get('{0}:{1}'.format(ip, port))
+
 
+class ManagerClientMockup(ASDManagerClient):
+    """
+    ASD Manager Client used by the unittests
+    """
+    test_results = {}
```

Comment:
Cannot be moved to init
I have tried and tested this and it does not work
This is because our DataList queries create a new object using the guids they query
This results in the init being triggered again and thus re-instantiating the mocked client
So moving this to the init, results in an empty dict before each iteration of `checkup_maintenance_agents`, which is unwanted behavior